plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2027,18 +2059,28 @@ public void registerCoreMembers() {
         // @Syntax invisible [<entity>] (state:true/false/toggle)
         // @Required 1
         // @Stable unstable
-        // @Short Makes an NPC or entity go invisible (Note: use '/npc playerlist' to make it work on NPCs!)
+        // @Short Makes an NPC or entity go invisible
         // @Author aufdemrand, mcmonkey
         // @Group entity
         //
         // @Description
-        // TODO: Document Command Details
+        // For non-armor stand entities, applies a maximum duration invisibility potion.
+        //
+        // For armor stands, toggles them invisible.
+        //
+        // NPCs can't be made invisible if not added to the playerlist.

```

Comment:
The trait automatically adds the playerlist status, which should be made clear here.